### PR TITLE
KG - Display SRID When Managing Service Request in Catalog (1.7.0)

### PR DESCRIPTION
**Pull request by kyle-glick**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -31,9 +31,13 @@ def show_welcome_message current_user
     raw(returning_html)
   end
 
-  def show_friendly_ssr_id ssr
-    unless ssr.nil?
-      "Editing ID: #{ssr.service_request.protocol.id}-#{ssr.ssr_id}"
+  def protocol_id_display protocol_id, sub_service_request, service_request
```
Comment (wtholt) [inline app/helpers/application_helper.rb]: Per style guide

     def protocol_id_display(protocol_id, sub_service_request, service_request)

File: Gemfile.lock [hunk 2]
```diff
@@ -165,7 +165,7 @@ GEM
     equivalent-xml (0.6.0)
       nokogiri (>= 1.4.3)
     erubis (2.7.0)
-    eventmachine (1.0.8)
+    eventmachine (1.2.0.1)
```
Comment (jwiel86) [inline Gemfile.lock]: Hey!  Why was this gem updated?

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -31,9 +31,13 @@ def show_welcome_message current_user
     raw(returning_html)
   end
 
-  def show_friendly_ssr_id ssr
```
Comment (jwiel86) [inline app/helpers/application_helper.rb]: We're curious, why did you remove this method?

**Comment by kyle-glick** [comment]
@jwiel86 
I updated eventmachine because my machine was continuously failing to install the current version we are trying to run. 
The show_friendly_ssr_id method wasn't removed, but rather renamed and modified to fit the new configuration.
